### DISCUSSION TITLE
FrameworkConfig not found in symfony/config

### DISCUSSION THREAD

**Discussion by tacman** [symfony/symfony#53980] [Q&A]
Where is Symfony\Config\FrameworkConfig?  I've followed the instructions, but neither the IDE nor code is finding it.

https://symfony.com/doc/current/reference/configuration/framework.html#workflows

![image](https://github.com/symfony/symfony/assets/619585/dfe13727-cae1-4f34-9a0a-52bd8de65f45)

I've explicitly installed symfony/config, but even in that component I can't find the file.

![image](https://github.com/symfony/symfony/assets/619585/e2abdc0e-548a-44cf-805b-39380fb08522)

I need to modify the framework within my bundle, I'm pretty sure it used to work but not sure what changed or what I'm doing wrong. Thanks!

**Comment by ahundiak** [comment]
It is a generated file. Look under var/cache/dev/Symfony. The only real way to "modify" it would be to add to the framework's configuration tree.

**Comment by tacman** [comment]
Thanks.  Perhaps I'm using it incorrectly, in a way that used to work but no longer does.

I want to my bundle to dynamically add the workflows to framework, presumably in the compilerPass.  Specifically, I'm going to iterate through all the classes that implement an interface (or maybe are tagged), and get the workflow configuration there.  But I'm stuck on how to inject my new workflows into the framework.

I used to autowire a class that did this modification 

```
        $builder->autowire(ConfigureFromAttributesService::class)
            ->setAutoconfigured(true)
            ->setPublic(true)
        ;
```

But now it appears that class isn't being called.

I still struggle with the compilerPass and prependExtension.  Am I approaching this the right way?  In short, I'm trying to add a workflow from within my bundle.  Thanks.
